clang/include/clang/Driver/Options.td

```diff
@@ -5337,9 +5337,13 @@ def mmadd4 : Flag<["-"], "mmadd4">, Group<m_mips_Features_Group>,
 def mno_madd4 : Flag<["-"], "mno-madd4">, Group<m_mips_Features_Group>,
   HelpText<"Disable the generation of 4-operand madd.s, madd.d and related instructions.">;
 def mmsa : Flag<["-"], "mmsa">, Group<m_mips_Features_Group>,
-  HelpText<"Enable MSA ASE (MIPS only)">;
+  Visibility<[ClangOption, CC1AsOption]>,
```

Comment:
We only need one codegen option.

Reply:
Did you mean delete this line of code?
If delete, would occur:
```
$ ./build/bin/clang -target mipsel-unknown-linux-gnu -Wa,-mmsa ./1.s
clang -cc1as: error: unknown argument: '-mmsa'
```

Reply:
See how --crel --no-crel is handled. Add a bool variable and only add `-mmsa` when the variable is true.

Reply:
Hi @MaskRay, I submitted changes, could you help review again? Thanks.